docs/connectors.md

```diff
@@ -18,12 +14,65 @@ Give the issue a `+1` if you really need that connector. The upvote will help th
 particular connector. If you find that an issue hasn't been created for your data store, please create a new issue in
 the Conduit repo.
 
-## Connectors
+## More about Connectors
+
+### Built-in connectors
+
+To help developers bootstrap pipelines much more quickly,
+Conduit ships with several built-in connectors by default.
+This includes the Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.
+
+If you want to create your own Conduit binary with a different set of built-in connectors, you can:
+
+- Find an [existing standalone connector](#the-list) that you would like to build-in
+or you can [create your own with the Conduit Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk)
```

Comment:
our simplest way to create a connector so far is using the template, so maybe we could link this https://conduit.io/docs/connectors/building here instead, which mentions both the SDK and template.

Reply:
Oh, I didn't know this, thank you for sharing! 🙏🏾